src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -416,6 +420,16 @@
   }
 }
 
+.pf-c-dropdown__menu-item-description {
+  font-size: var(--pf-c-dropdown__menu-item-description--FontSize);
+  color: var(--pf-c-dropdown__menu-item-description--Color);
+}
+
+.pf-c-dropdown__menu-item-title {
+  display: flex;
```

Comment:
Hmm... this is going to make text in the item in a regular dropdown (no description) and an item with a description display differently. I think this flex styling needs to be scoped to when `.pf-m-icon` is applied to the top level item. Though I see `.pf-m-icon` is missing from the item in these description examples. Is that intentional?

One way is we could move `.pf-m-icon` from `.pf-c-dropdown__menu-item` to `.pf-c-dropdown__menu-item-title` when a description is used, and update `.pf-c-dropdown__menu-item.pf-m-icon { display: flex; align-items: center;}` to include `.pf-c-dropdown__menu-item-title.pf-m-icon`.

You could also do something like this which might be easier on the react side:

```
li // has icon
  a.pf-c-dropdown__menu-item.pf-m-description.pf-m-icon // leaves .pf-m-icon on the top level menu-item with or without a description
    div.pf-c-dropdown__menu-item-title // or whatever you end up calling it
      span.pf-c-dropdown__menu-item-icon
        // some icon
      the dropdown item text
    div.pf-c-dropdown__menu-item-description
      the dropdown item description

li // no icon
  a.pf-c-dropdown__menu-item.pf-m-description
    div.pf-c-dropdown__menu-item-title // or whatever you end up calling it
      the dropdown item text
    div.pf-c-dropdown__menu-item-description
      the dropdown item description
```

And then the CSS

```
.pf-c-dropdown__menu-item {
  &.pf-m-description {
    flex-direction: column; // fixes the direction when `.pf-m-icon` is applied
       
    &.pf-m-icon {
      .pf-c-dropdown__menu-item-main {
        display: flex;
        align-items: center;
      }
    }
}
```